src/dataconnect/fileUtils.spec.ts

```diff
@@ -127,81 +129,254 @@ describe("getPlatformFromFolder", () => {
       const platform = await getPlatformFromFolder(c.folderName);
       expect(platform).to.equal(c.output);
     });
-    afterEach(() => {
-      mockfs.restore();
-    });
   }
+  afterEach(() => {
```

Comment:
Should this be moved out? It seems valuable to keep it where it was before and reset the mocks in between tests

Reply:
Ah, actually, this seems correct.